build.gradle

```diff
@@ -427,9 +427,8 @@ tasks.withType(Test) {
 task vendorShadowJar(dependsOn: shadowJar) {
     description "vendors jar dependencies"
 
-    String vendorPathPrefix = "vendor/jar-dependencies"
-    String projectGroupPath = project.group.replaceAll('\\.', '/')
-    File projectJarFile = file("${vendorPathPrefix}/${projectGroupPath}/${project.name}/${project.version}/${project.name}-${project.version}.jar")
+    String vendorPathPrefix = "vendor/jar-dependencies/co/elastic"
```

Comment:
review note: made a minimal change to shorten the path. We may directly place into vendor but we are using `jar-dependencies` (in gemspec file) as a required path and include jars as a standard in other Java based plugins.